### ISSUE TITLE
[Report] Issues with Intel Iris iGPU Laptop

### ISSUE THREAD

**Issue by Ew4n1011** [zweiler2/HoloISO_Nvidia#6]
## Your Issue:
```
*Various problems with Intel Laptop, listed below:*
  - Gamemode (mangohud) screen goes black when I move the mouse, works fine with keyboard/controller though
  - 8bitdo controller doesn't work, only works if I put it on switch mode
  - Selected decky on install, but its nowhere to be found
  - Also selected emudeck on install, but can't find it anywhere neither
```

## Your device's full specs
```
CPU: Intel 11th Gen Core i5-1135G7 @ 2.40 GHz
GPU: Intel Xe Graphics (iGPU)
```

## Your OS version (`cat /etc/os-release | grep VARIANT_ID | cut -d '"' -f 2`):
```
snapshot20221222.1749
```

## (Only for SteamOS Session loading issues) Have you tried launching `gamescope-session` from terminal? If yes, provide a log.
```
Unable to run via terminal, gives error on line 254: 6246 killed
```

## Your installation type:
```
deckperience
```

## Have you updated your installation recently?
```
Just installed today
```

**Comment by zweiler2** [comment]
Only the mangohud overlay goes black when you move the mouse or the whole screen?  
Have you tried removing lines 8-12 from /usr/bin/gamescope-session?  
Which 8bitdo controller exactly?  
I'll look into the other two issues.

**Comment by Ew4n1011** [comment]
Yep, it still does it, I've commented those lines out but it still acts the same
As for the controller, its the [8bitdo Ultimate Bluetooth Controller](https://download.8bitdo.com/Manual/Controller/Ultimate/Ultimate-Bluetooth-Controller.pdf)

The controller is connected using the 2.4 GHz dongle

Also another issue which I forgot to mention and just realised after rebooting (sorry) is that it doesn't show as a boot option, I have to manually select the EFI file from my bios to boot it.

The BIOS is a HP BIOS Setup Utility if that helps

**Comment by Ew4n1011** [comment]
Just realised I didn't answer one of your questions, the entire screen went black and the only way to return it was by pressing a key on my keyboard or on my controller
